### DISCUSSION TITLE
Need Assistance with Accessing My GitHub Account and Problem with 2FA

### DISCUSSION THREAD

**Discussion by AyseAyhann** [community/community#111251] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Some time ago, I set up my GitHub account to be accessed with two-factor authentication, but I no longer have access to either my passkey or my 2FA app password because I deleted the 2FA application. I want to access my account. I can prove that the account belongs to me by demonstrating that the LinkedIn account I added to my bio belongs to me, and also the profile picture is mine. I know the email address and password for the account, but I don't have any passkey for 2FA, although I can prove that the account belongs to me. Please assist me; I have reviewed the documents on your website, but there is no solution. There is a suggestion to remove my email, but I do not want to do that. Legally, I can prove that the account belongs to me and is in my name. Can you help me access my account?

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation

Acces the [Github Support](https://support.github.com/)

Click on My Ticket  - New Ticket - And describe your situation

**Comment by AyseAyhann** [comment]
Not, it is not working, the automatic replying program is not working.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @AyseAyhann! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, I'm really sorry to hear that you're having trouble accessing your account right now. Our support docs actually address [Recovering your account if you lose your 2FA credentials](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials). I recommend checking out that article and following the advice there. If all else fails you can try contacting [Support](https://support.github.com/contact) (or emailing support@github.com directly) to see if there's any possibility they can help.

I've gone ahead and moved this to the correct category for you. Good luck! 🍀
